scripts/public-circleci-lib-dist.js

```diff
@@ -2,36 +2,32 @@
 const release = require('../utils/release');
 const fs = require('fs-extra');
 
-async function getDistFiles() {
-  const distfiles = [];
-
-  // Don't override the dist/package.json if it's already bundled.
-  if (!fs.existsSync('./dist/package.json')) {
-    distfiles.push(fs.copy('./package.json', './dist/package.json'));
+async function copyBestPath(bestPaths, targetPath, overwrite = true) {
+  if (!overwrite
+     && fs.existsSync(targetPath)) {
+    console.log(`Skipping ${bestPaths} as ${targetPath} has targeted file.`);
```

Comment:
Should there be `${bestPaths.join(' ')}` here?

Reply:
Good spot!